src/app/public/modules/lookup/lookup.component.scss

```diff
@@ -93,6 +105,11 @@ sky-input-box .sky-lookup {
 
 @include sky-theme-modern {
 
+  .sky-lookup-inset-search-icon {
+    font-size: 24px;
```

Comment:
Should we be using `$sky-theme-modern-font-heading-2-size` or `$sky-theme-modern-text-size-400`?

Reply:
Input box wasn't using a variable, so I just did the same:
https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/input-box/input-box.component.scss#L114

Reply:
@Blackbaud-AdamFunderburk - should we use the variable here? Or just hard code this- this boils down to if we want that to update if we ever update the design tokens. Input box might be missing the variable as well lol.

Reply:
This code has been moved to another PR: https://github.com/blackbaud/skyux-forms/pull/266